R/cancelOrders.R

```diff
@@ -97,27 +98,64 @@
 #'
 
 cancelOrders <-
-  function(marketId, betIds = NULL, sizeReductions = NULL, suppress = FALSE, sslVerify = TRUE) {
+  function(marketId = NULL, betIds = NULL, sizeReductions = NULL, customerRef = NULL, suppress = FALSE, sslVerify = TRUE) {
```

Comment:
I think it's super dangerous here to have `marketId = NULL`. If the end user simply executes `cancelOrders()`, all bets across all markets are cancelled.

Back in the mists of time, I think I deliberately didn't set `marketID = NULL` from a risk perspective.

Reply:
Oops, yes, the show() shouldn't be there.

Good point on the risk around marketId = NULL. My thought here was that it is not a required parameter by the API but agree it is much safer to require the user to spell out that they want to cancel all orders.